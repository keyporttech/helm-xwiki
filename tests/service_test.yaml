suite: test service
templates:
  - service.yaml
  - deployment.yaml
tests:
  - it: should set values properly
    set:
      service.type: ClusterIP
      service.externalPort: 80
      service.internalPort: 8080
      service.name: tcp
    release:
      name: release
    asserts:
      - isKind:
          of: Service
      - matchSnapshot:
          path: spec
  - it: should have same labels as Deployment
    set:
      service.type: ClusterIP
      service.externalPort: 80
      service.internalPort: 8080
      service.name: tcp
    release:
      name: release
    asserts:
      - equal:
          path: spec.selector.app
          value: release-xwiki
      - equal:
          path: spec.selector.matchLabels.app
          value: release-xwiki
        template: deployment.yaml
      - equal:
          path: spec.template.metadata.labels.app
          value: release-xwiki
        template: deployment.yaml