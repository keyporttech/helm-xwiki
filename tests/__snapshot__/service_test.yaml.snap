should set values properly:
  1: |
    ports:
    - name: tcp
      port: 80
      protocol: TCP
      targetPort: 8080
    selector:
      app: release-xwiki
    sessionAffinity: ClientIP
    type: ClusterIP
